### DISCUSSION TITLE
Selección de tema para proyecto

### DISCUSSION THREAD

**Discussion by jpablocolorado** [rramosp/ai4eng.v1#84] [00 - General]
Buenos días profe. Nuestro grupo tiene en mente varios temas para el proyecto de investigación. Todos están en Kaggle, pero queremos saber su opinión acerca de cual podríamos escoger o que hay que tener en cuenta para dicha escogencia.

estos son los proyectos en mente
1- https://www.kaggle.com/datamarket/venta-de-coches
2- https://www.kaggle.com/angiemantilla/datostfg
3- https://www.kaggle.com/arm110/precios-de-exportaci-n-e-importaci-n-de-productos
4- https://www.kaggle.com/malgoro39/predicci-n-de-destinos-airbnb
5- https://www.kaggle.com/c/pakdd-cup-2014/leaderboard
6- https://www.kaggle.com/neuromusic/avocado-prices/tasks

**Comment by Felipe-RA** [comment]
Saludos jpablocolorado

La mayoría de los enlaces que provees son datasets, te recomiendo que cuando busques lo hagas a través de la pestaña de **Competiciones** pues es allí donde estan definidos los proyectos bien definidos (Las métricas, cuál es la tarea predictiva,etc.)

La número 5,  sobre las fallas en componentes de ASUS me parece apropiada para el alcance del curso.

Como siempre, te invito a que te pases por las sesiones sincrónicas para que lo hablemos más a detalle

Hasta pronto!
